web/sass-files/sass/partials/_modal.scss

```diff
@@ -29,7 +29,7 @@
 		margin-right: auto;
 	}
 	.modal-body {
-		overflow: auto;
+		overflow: scroll;
```

Comment:
this will override with https://github.com/mattermost/platform/blob/master/web/react/components/post.jsx#L16